stylus/components/button.styl

```diff
@@ -0,0 +1,30 @@
+// Buttons of any kind
+.coz-btn
+    box-sizing border-box
+    display inline-block
+    vertical-align top
+    margin 0 em(8px)
+    border 1px solid grey-09
+    height: em(43px)
```

Comment:
Syntactic purism, but we should definitely choose between colons or no-colons, and if we pefer values aligned or not. Any preference on this? (previously we said no-colons and aligned https://github.com/cozy/cozy-guidelines#css--stylus, but we can updat this guide too).

/poke @gregorylegarec, @goldoraf, @CPatchane, @nono about this one